jballerina.java.arrays-ballerina/build.gradle

```diff
@@ -218,9 +217,15 @@ task ballerinaPublish {
         } else {
             throw new InvalidUserDataException("Central Access Token is not present")
         }
```

Comment:
Check indentation

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-jballerina.java.arrays/pull/86/commits/b18d6bf8244123b462d138612a487209a35e2d1e